serialization/json-ld.md

```diff
@@ -0,0 +1,41 @@
+# JSON-LD
+JSON-LD is a JSON-based format to encode RDF graphs. Its documentation can be found [here](https://www.w3.org/TR/json-ld11/).
+
+The JSON-LD is an RDF format and follows the serialization rules of the [rdf SPDX serialization specification](rdf.md).
+
+## XCollection
+
+The following XCollection properties are mapped to JSON-LD syntax specifications.
+Any properties not listed below should be serialized as part of the XCollection element itself within
+the JSON-LD serialized data.
+
+### namespaceMap
+
+The namespaceMap uses the [term to IRI mapping](https://www.w3.org/TR/json-ld11/#example-11-term-expansion-from-context-definition) in the [JSON-LD context](https://www.w3.org/TR/json-ld11/#the-context).
+
+### element
+
+The [graph objects](https://www.w3.org/TR/json-ld11/#graph-objects) `@graph` lists the elements for the XCollection.
+
+The RDF graph of an instance of the SPDX model shall contain all Element nodes (i.e. objects that are subclasses of Element) as a list on top-level under the "@graph" key.
+This means that all references to Element nodes have to use the URI of the referenced Element.
+Inlining/Embedding of Element nodes into other nodes is not allowed.
+On the other hand, non-Element nodes (like those of type "ExternalReference" or similar complex data classes) have to be inlined (TODO: we may want to make an exception for CreationInfo, depending on the outcome of the surrounding discussion).
```

Comment:
It also needs calling out explicitly here that each embedded non-Element node MUST have an explicit id. This is true for ALL serialization formats. 
The serialized embedding within the json-ld object structure is a method of compaction to make the content easier to read (and process by some parsers) but it does not inherently represent the native deserialized logical content which is a graph where EVERY class instance (Element nodes and non-Element nodes) are separate nodes in the graph connected using the globally unique id of each node which MUST be present on all nodes.

There is a possible serialization compaction approach where we COULD (though I would not recommend it) assert that in situations where a producer is minting new elements to be directly serialized without creation anywhere internally first which would require ids already be assigned they could use blank node ids (like has been discussed for CreationInfo) within a specific serialization instance. These blank nodes would then be REQUIRED to be skolemized to create full globally unique ids (like done for CreationInfo) upon any deserialization. This could potentially save some space within the serialized content but it would likely be minimal and the extra complexity in handling and potential for misinterpretation makes this approach a questionable option to pursue.

Reply:
Same as above - i just copy/pasted what was there before.